src/issue/issue.service.ts

```diff
@@ -0,0 +1,76 @@
+import { Injectable, NotFoundException, InternalServerErrorException, Logger } from '@nestjs/common';
+import { CreateIssueDto } from './dto/create-issue.dto';
+import { UpdateIssueDto } from './dto/update-issue.dto';
+import { Repository } from 'typeorm';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Issue } from './entities/issue.entity';
+
+@Injectable()
+export class IssueService {
+  constructor(
+    @InjectRepository(Issue)
+    private issueRepository: Repository<Issue>,
+  ) {}
+
+  async createIssue(createIssueDto: CreateIssueDto): Promise<Issue> {
```

Comment:
remove try and catch from all the code, nestjs have a system for detecting error that is better than try / catch

Reply:
ok